site/content/community/users.md

```diff
@@ -19,7 +19,13 @@ considering their efforts before starting your own!
 
 | Name                   | Description                                        |
 |:-----------------------|----------------------------------------------------|
-| [http-wasm-host-go][3] | Serves HTTP handlers implemented in [http-wasm][4] |
+| [http-wasm-host-go][3] | serves HTTP handlers implemented in [http-wasm][4] |
```

Comment:
Two started with lowercase vs 1 uppercase, so went ahead and made consistent to start with lowercase